### ISSUE TITLE
Built failed on the latest `master` branch

### ISSUE THREAD

**Issue by IceCodeNew** [cloudflare/cfnts#47]
Built failed on the latest `master` branch commit https://github.com/cloudflare/cfnts/commit/3d9c673e1b7abbad1bd691ef7c1608582e8371a6

* Log
```
Cloning into '/git/cfnts'...
Updating crates.io index
error: failed to select a version for the requirement `zeroize = "^0.5"`
candidate versions found which didn't match: 1.3.0, 1.2.0, 1.1.1, ...
location searched: crates.io index required by package `miscreant v0.4.2` ...
which is depended on by `cfnts v2019.6.0 (/git/cfnts)`
```

**Comment by wbl** [comment]
Could you please describe the complete series of steps? It sounds like you cloned this repo, cd in, and then ran cargo update. Is that correct? I'll try to take a look sometime this week: patches welcome if you figure out what busted earlier.

**Comment by IceCodeNew** [comment]
> Could you please describe the complete series of steps? It sounds like you cloned this repo, cd in, and then ran cargo update. Is that correct?

Yep. the commands I issued was just `git clone https://github.com/cloudflare/cfnts.git` & `cargo update --verbose`

**Comment by IceCodeNew** [comment]
> Which version of cargo? I didn't have the same problem (and sorry for letting this linger)

The version comes with rust 1.51.0.
As it is already 6mo ago and all CI logs are emptied. I can only tell that the cargo version should be the same as the version install by the rustup default install then.
